.travis.yml

```diff
@@ -4,7 +4,10 @@ rbenv:
 addons:
   postgresql: 9.6
   chrome: stable
-dist: trusty
+dist: xenial
```

Comment:
This is a good change - this cleared the errors we were getting in Travis CI about the incorrect chrome version